cmake/configs/posix_sitl_default.cmake

```diff
@@ -144,6 +144,12 @@ set(config_module_list
 	#examples/hwtest
 )
 
+# Additional Flight Tasks
+# (on top of the core tasks that mc_pos_ctrl needs)
+set(flight_tasks_additional
+		Orbit
```

Comment:
Can we invert this? It would be better to have full support everywhere, but the ability to strip it down for special constrained configurations.

Reply:
From a company perspective, it is more attractive to have an official way to add tasks that are part of the company feature set and adds value to the product. For instance, Manual, Altitude, Position, Auto and Offboard can be considered as core tasks and definitely need to be fully supported on the PX4 platform. However, there are other tasks that companies require to have but are very unique. Those tasks do not require upstream support.

Reply:
Yes I think we can still make that possible.

I'm trying to ensure the pieces that are upstream receive as much support and testing as possible and try to provide a consistent experience across all boards. Not to mention documentation and QGC support.

How about...
 - all upstream Flighttasks included by default
 - a mechanism to extend it with your own custom FlightTasks
 - if necessary a mechanism to exclude built in FlightTasks

Reply:
>How about...
>all upstream Flighttasks included by default
>a mechanism to extend it with your own custom FlightTasks
>if necessary a mechanism to exclude built in FlightTasks

Sounds reasonable:+1: I'll make the adjustments